contracts/utils/Nonces.sol

```diff
@@ -13,7 +13,7 @@ abstract contract Nonces {
     mapping(address account => uint256) private _nonces;
 
     /**
-     * @dev Returns an the next unused nonce for an address.
+     * @dev Returns the current nonce for an address.
```

Comment:
The existing description is accurate, but has a grammar error. This function returns the "next unused nonce", i.e. the nonce that will work when passed to `_useCheckedNonce`.

```suggestion
     * @dev Returns the next unused nonce for an address.
```